compact/tof_barrel.xml

```diff
@@ -107,18 +115,18 @@
       </module>
       <layer module="BarrelTOF_Module1" id="1" vis="TOFLayerVis">
         <barrel_envelope
-          inner_r="BarrelTOF_rmin-BarrelTOF_rOffset"
-          outer_r="BarrelTOF_rmax+BarrelTOF_rOffset"
-          z_length="BarrelTOF_length"/>
-        <rphi_layout phi_tilt="BarrelTOF_Module_tiltangle" nphi="BarrelTOF_Module_nphi" phi0="0.0" rc="BarrelTOF_rmin" dr="0.0*mm"/>
-        <z_layout dr="0.0*mm" z0="15.0*cm" nz="BarrelTOF_Module_nz"/>
+          inner_r="BarrelTOF_rmin-1*mm"
```

Comment:
You cannot go less than `rmin` at all, not even a millimeter. That will(!) result in conflicts with other volumes.

Reply:
Fixed